$OpenBSD: patch-mailnews_base_search_src_nsMsgSearchTerm_cpp,v 1.1 2014/01/11 08:35:15 landry Exp $

Fix clang compile failure in nsMsgSearchTerm.cpp by using an empty string
instead of a null pointer, from bug #885538
https://hg.mozilla.org/comm-central/rev/89bbcd5a2293

--- mailnews/base/search/src/nsMsgSearchTerm.cpp.orig	Tue Jan  7 23:29:40 2014
+++ mailnews/base/search/src/nsMsgSearchTerm.cpp	Tue Jan  7 23:30:53 2014
@@ -197,7 +197,7 @@ nsresult NS_MsgGetStringForAttribute(int16_t attrib, c
     }
   }
   if (!found)
-    *string = '\0'; // don't leave the string uninitialized
+    *string = ""; // don't leave the string uninitialized
 
   // we no longer return invalid attribute. If we cannot find the string in the table,
   // then it is an arbitrary header. Return success regardless if found or not
